src/backend/expungeservice/app.py

```diff
@@ -1,12 +1,12 @@
 import importlib
```

Comment:
This is currently an unused import... Perhaps we can remove it at this time?

Reply:
yep.